src/float/surface_transceiver/surface_transceiver.ino

```diff
@@ -69,13 +68,13 @@ void loop() {
     Serial.println("Getting serial command...");
     String command = Serial.readString();
 
-    if (command.startsWith("submerge")) {
+    if (command.equals("submerge")) {
```

Comment:
Behavior change

Reply:
True but, currently a `submerge foo bar` would be valid which seems like unecssary.